packages/paypal-commerce-integration/src/paypal-commerce-alternative-methods/paypal-commerce-alternative-methods-payment-initialize-options.ts

```diff
@@ -127,6 +127,11 @@ export interface PaypalCommerceRatePay {
      */
     legalTextContainer: string;
 
+    /**
+     * Function that show loading spinner overlay
```

Comment:
First of all this callback should be optional, so if the dev wont do anything when ratepay will be in pending state, it is theirs responsibility.

I may suggest to change `isLoading` to `isPending`.

And update the method description with something more meaninfull, because you don't know what will be in those callback